test/metabase/public_settings/premium_features_test.clj

```diff
@@ -40,8 +41,17 @@
                 :trial    false}))
 
 (deftest fetch-token-status-test
-  (tt/with-temp User [user {:email "admin@example.com"}]
-    (let [token "fa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3e"]
+  (tt/with-temp User [_user {:email "admin@example.com"}]
+    (let [token       "fa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3ebfa3e"
+          print-token "fa3e...fa3e"]
```

Comment:
nit: maybe use a test token that has different first four & last four characters, just for the extra validation that truncation is happening correctly?